client/src/components/app-mount-notifier.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { render, waitFor } from '@testing-library/react';
+import { Provider } from 'react-redux';
+
+import { i18nextCodes } from '../../../config/i18n/all-langs';
+import AppMountNotifier from './app-mount-notifier';
+import { createStore } from '../redux/createStore';
+import i18nTestConfig from '../../i18n/configForTests';
+
+jest.mock('react-ga');
+jest.unmock('react-i18next');
+
+type Language = [string, string];
+
+const store = createStore();
+
+// Create a nested array for languages
+const languages = Object.keys(i18nextCodes).map(
+  /* eslint-disable @typescript-eslint/ban-ts-comment, @typescript-eslint/no-unsafe-return */
+  // @ts-ignore
+  // TODO: convert `all-langs.js` to TypeScript
+  (key): Language => [i18nextCodes[key], key]
+);
+
+describe('AppMountNotifier', () => {
+  const setup = (lang: string) => {
+    i18nTestConfig.language = lang;
+
+    render(
+      <Provider store={store}>
+        <I18nextProvider i18n={i18nTestConfig}>
+          <AppMountNotifier render={() => <p>App content</p>} />
+        </I18nextProvider>
+      </Provider>
+    );
+  };
+
+  test.each(languages)(
+    'should set the lang attribute to %s if the language is %s',
+    async langCode => {
+      setup(langCode);
```

Comment:
The two `%s` represent the items in the subarray. The first `%s` is the language code, and the second is the language name.

<img width="670" alt="Screen Shot 2021-07-03 at 07 52 58" src="https://user-images.githubusercontent.com/25715018/124339194-e49e5c80-dbd6-11eb-8054-9aaec63ea6ab.png">